src/OnlineSales/Entities/EmailSchedule.cs

```diff
@@ -0,0 +1,26 @@
+﻿// <copyright file="EmailSchedule.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.Json.Serialization;
+
+namespace OnlineSales.Entities
+{
+    public class EmailSchedule : BaseEntity
+    {
+        /// <summary>
+        /// Gets or sets the JSON based schedule assigned for the email group.
```

Comment:
May be let's put some schedule sample here in the comments, so we see what types of schedules are possible?

Reply:
Updated comments